dev/oci_env_integration/oci_env_configs/dab.compose.env

```diff
@@ -1,7 +1,7 @@
 COMPOSE_PROFILE=galaxy_ng/base:galaxy_ng/dab
 COMPOSE_PROJECT_NAME=ci-dab
 
-DEV_SOURCE_PATH=galaxy_ng
+DEV_SOURCE_PATH=galaxy_ng:pulp_ansible
```

Comment:
does DAB compose need pulp_ansible as dev source?

Reply:
Nope, we just need a new release (hopefully today) of pulp_ansible before I can take this out.